synnax/pkg/distribution/channel/ontology.go

```diff
@@ -43,14 +43,15 @@ func OntologyIDsFromChannels(chs []Channel) []ontology.ID {
 var _schema = &ontology.Schema{
 	Type: OntologyType,
 	Fields: map[string]schema.Field{
-		"key":       {Type: schema.Uint32},
-		"name":      {Type: schema.String},
-		"node_key":  {Type: schema.Uint32},
-		"rate":      {Type: schema.Float64},
-		"is_index":  {Type: schema.Bool},
-		"index":     {Type: schema.String},
-		"data_type": {Type: schema.String},
-		"internal":  {Type: schema.Bool},
+		"key":        {Type: schema.Uint32},
+		"name":       {Type: schema.String},
+		"node_key":   {Type: schema.Uint32},
+		"rate":       {Type: schema.Float64},
+		"is_index":   {Type: schema.Bool},
+		"index":      {Type: schema.String},
+		"data_type":  {Type: schema.String},
+		"internal":   {Type: schema.Bool},
+		"expression": {Type: schema.String},
```

Comment:
I think `virtual` is still missing here. Can you make sure that all fields have been added?

Reply:
Do i add one to pass the slice of requires keys?